packages/pipeline/migrations/1543446690436-CreateDexTrades.ts

```diff
@@ -0,0 +1,41 @@
+import { MigrationInterface, QueryRunner, Table } from 'typeorm';
+
+const dexTrades = new Table({
+    name: 'raw.dex_trades',
+    columns: [
+        { name: 'source_url', type: 'varchar', isPrimary: true },
+        { name: 'tx_hash', type: 'varchar', isPrimary: true },
+
+        { name: 'tx_timestamp', type: 'bigint' },
+        { name: 'tx_date', type: 'varchar' },
+        { name: 'tx_sender', type: 'varchar(42)' },
+        { name: 'smart_contract_id', type: 'bigint' },
```

Comment:
Why do we have a smart_contract_id and a smart_contract_address? Isn't the address a unique ID?

Reply:
Take a look at https://bloxy.info/api_methods#dex. Since I'm not sure exactly what we will need, I'm just storing all the data they give us for now.

It looks like there might be a one-to-one relationship between `smart_contract_id` and `smart_contract_address` but I can't be certain. I can imagine it being the case that the address could change e.g. if a new version or hotfix for the contract is released.

Are you okay with keeping it for now until we get some clarification from the Bloxy team?